### ISSUE TITLE
Issues converting SAM to BAM

### ISSUE THREAD

**Issue by scuba-sarah** [course-fish546-2023/assistance-public#14]
See screenshot. I'm assuming this is not what I want?

![Screenshot 2023-05-05 145413](https://user-images.githubusercontent.com/122320648/236575341-9d3b0173-9596-4915-bea2-a6ab8bcc0b41.png)

**Comment by sr320** [comment]
try changing sam to `/home/shared/8TB_HDD_01/sr320/github/steven-crabby-duck/output/F143_cgigas.sam`

**Comment by scuba-sarah** [comment]
Nah, I got the same issue.

**Comment by sr320** [comment]
url for Rmd file and line #?

**Comment by scuba-sarah** [comment]
Sorry, I just pushed.
https://github.com/course-fish546-2023/sarahyerrace-coursework/blob/main/assignments/code/06_Alignment_Data.Rmd
line 116

**Comment by sr320** [comment]
I just ran - no issues

see https://d.pr/i/orVouR

**Comment by scuba-sarah** [comment]
yeah it'll run for a while then stop for me. And then the next chunk won't run either

**Comment by sr320** [comment]
running code in your repo.... https://d.pr/i/EfsZQ1 no errors. Are you running on raven?

**Comment by scuba-sarah** [comment]
Yes
